src/utils.ts

```diff
@@ -160,6 +166,17 @@ export function humanReadableDate(timestamp: number): string {
 	return new Date(timestamp).toLocaleString("sv");
 }
 
+export function getLogString(data: Metafile | Searchee, color: ChalkInstance) {
+	if (data instanceof Metafile) {
+		return `${color(data.name)} ${chalk.dim(`[${data.infoHash.slice(0, 8)}...]`)}`;
+	}
```

Comment:
is this instanceof check actually necessary? I would think duck typing should just work here

Reply:
It won't handle data.path without this check. It still thinks it could be Searchee after.